src/docker/Images.ts

```diff
@@ -10,6 +10,12 @@ export interface DockerImage extends DockerObject {
     readonly Size?: number;
 }
 
+export interface ImageInspectionContainers {

```

Comment:
Slightly more interesting change...defined and exported this here so it can be referenced in `DockerodeUtils.ts`